spec/requests/requests_spec.rb

```diff
@@ -289,7 +289,7 @@
 
       FactoryBot.create(:classification_department_with_tags)
 
-      t = Classification.where(:description => 'Department', :parent_id => 0).includes(:tag).first
+      t = Classification.where(:description => 'Department', :parent_id => nil).includes(:tag).first
```

Comment:
```suggestion
      t = Classification.is_category.includes(:tag).find_by(:description => 'Department')
```

Reply:
or `Classification.is_category` -> `Category` (it is a default scope here)

Reply:
yes. things get a little tricky with Category since a different object will be returned and specs compare and get a different value.